src/silx/gui/plot/backends/BackendOpenGL.py

```diff
@@ -1135,6 +1138,7 @@ def addMarker(
             text,
             color,
             symbol,
+            int(symbolsize + 0),
```

Comment:
Cast to int is not needed, a `float` is used internally:

```suggestion
            symbolsize,
```